Templates/CRMs/Zoho CRM/Zoho CRM to JSON - Products/long-description.md

```diff
@@ -0,0 +1,28 @@
+This template process retrieves the list of products from **Zoho CRM** and writes the information as a **JSON** file into the local fileshare.
+
+Using this template, you can retrieve the list of products from Zoho and store it locally as JSON.
+
+![Template](assets/Zoho-CRM-to-JSON-Products.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- You have a refresh token for the Zoho API. One can be created using the corresponding template.
+- The client ID and client secret for Zoho are available.
+- It is known which domain your Zoho instance is in, e.g., EU or America.
+- The Frends agent has access to the folder into which the JSON file should be written.
+
+# Implementation and Usage Notes
+
+This template requires a refresh token for the Zoho API in order to work. The template will then use the refresh token to create the access token for accessing the API. The refresh token can be created using the "Zoho CRM - Exchange grant token for refresh token" template.
```

Comment:
The "then" in this paragraph is not needed, because there is no previous step.

Reply:
Nice catch, thanks!